src/components/Events/Events.js

```diff
@@ -80,19 +116,10 @@ class Events extends Component {
             updateFilters={this.updateFilters.bind(this)}
           />
         </div>
-        <EventsList
-          events={events}
-          filters={filters}
-        />
-        <div className={styles.loadMoreBtn}>
-          { isFetchingMoreEvents ?
-            <Loading /> :
-            <button type="button" onClick={this.loadMoreEvents.bind(this)}>
-              load more events
-            </button>
-          }
-
-        </div>
+        { isFetchingEvents ?
+          <div className={styles.loadingWrapper}><Loading /></div> :
```

Comment:
Because there are two components called loading in the same file can we change the name of the wrapper to be more specific?